build.gradle

```diff
@@ -61,6 +61,16 @@ pluginBundle {
     }
 }
 
+
```

Comment:
Can you add short code comment why we are doing this? It's very useful to grasp the implementation quickly, without navigating git history + github issue description.